packages/flame/lib/src/sprite_animation.dart

```diff
@@ -243,6 +246,20 @@ class SpriteAnimation {
   /// still image).
   bool get isSingleFrame => frames.length == 1;
 
+  /// A future that will complete when the animation completes.
+  ///
+  /// An animation is considered to be completed if it reaches its [isLastFrame]
+  /// and is not [loop]ing.
+  Future<void> get completed {
```

Comment:
When I read this property name I would assume it returns a `bool` instead of a `Future<void>`

Maybe a more descriptive name would fit? Something`whenCompleted`? What do you think @st-pasha @spydon

Reply:
This follows the same pattern as `mounted` etc where we also had this discussion, so I think this is fine.
A bool should have `is` or `has` prepended to it in most cases.

Reply:
I'm open to renaming but this follows exactly the same logic that `loaded` and `mounted` in `Component` have.

Reply:
Aight, that clears it up for me!